azure-communication-ui/build.gradle

```diff
@@ -35,7 +35,7 @@ buildscript {
             azure_calling_sdk_version = privateDropProps.getProperty('PRIVATE_DROP_VERSION', '""')
         }
         else {
-            azure_calling_sdk_version = '2.2.1'
+            azure_calling_sdk_version = '2.3.0-beta.1'
```

Comment:
Just double checking, since we plan this feature to go out for GA, can we use a stable GA calling sdk version?

Reply:
This PR will not be merged, only to show differences made for the feature.